docs/implementation-guide.md

```diff
@@ -55,131 +55,43 @@ We have only provided a brief overview of PHDI, Building Blocks, and the pipelin
-### Step 2: Install the Azure and GitHub CLI tools
-The az CLI is a command line tool provided by Microsoft for working with Azure. We will use it to authenticate your local machine with your organization's Azure environment. Follow [this guide](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) to install `az`. Confirm that the installation was successful by running `az --version`. If `az` is installed properly you should get a response similar to what is shown below.
+### Step 1: Ensure you have collected values for geocoding variables
 
-```bash
-❯ az --version
-azure-cli                         2.40.0
-
-core                              2.40.0
-telemetry                          1.0.8
-
-Extensions:
-account                            0.2.4
-
-Dependencies:
-msal                            1.18.0b1
-azure-mgmt-resource             21.1.0b1
+It is not recommended to exit the quick start script partway through so please have all values on hand when you run the script.
 
-Python location '/opt/homebrew/Cellar/azure-cli/2.40.0/libexec/bin/python'
-Extensions directory '/Users/myuser/.azure/cliextensions'
-
-Python (Darwin) 3.10.6 (main, Aug 30 2022, 04:58:14) [Clang 13.1.6 (clang-1316.0.21.2.5)]
+Required to use geocoding functionality:
+- `SMARTY_AUTH_ID` - Your SmartyStreet Authorization ID. More info on the Smarty geocoding service [here](https://www.smarty.com/pricing/us-rooftop-geocoding)
+- `SMARTY_AUTH_TOKEN` - Your SmartyStreet Authorization Token.
 
-Legal docs and information: aka.ms/AzureCliLegal
+### Step 2: Run the Quick Start Script in Azure Cloud Shell
+In this step we will work through Azure's [Workload Identity Federation](https://learn.microsoft.com/en-us/azure/active-directory/develop/workload-identity-federation) to grant your phdi-azure repo access to deploy the pipelines to your organization's Azure environment. We have provided a script to automate most of this process that we recommend you use. However, if you prefer to work through it manually you may follow [this guide](https://learn.microsoft.com/en-us/azure/active-directory/develop/workload-identity-federation-create-trust?pivots=identity-wif-apps-methods-azcli).
 
+Navigate to the [Azure Cloud Shell](https://shell.azure.com/).
 
-Your CLI is up-to-date.
+From your Azure account's command line:
+1. Clone this repository by running the following command:
+```bash
+git clone https://github.com/CDCgov/phdi-azure.git && cd phdi-azure
 ```
-
-Optionally, you may also install the GitHub CLI tool (`gh`) to enable automatically setting secrets in your repository. Follow [this guide](https://cli.github.com/manual/installation) to install the tool. Confirm that the installation was successful by running `gh --version`. If `gh` is installed properly you should get a response similar to what is shown below.
-
+2. Run the quick start script:
 ```bash
-❯ gh --version
-gh version 2.14.7 (2022-08-25)
-https://github.com/cli/cli/releases/tag/v2.14.7
+./quickstart.sh
 ```
```

Comment:
What do you think about simply having them run this?
`git clone https://github.com/CDCgov/phdi-azure.git && cd phdi-azure && ./quickstart.sh`

We could just say something like - "To download and run the quick start script, run the following command"